rollup.config.js

```diff
@@ -1,8 +1,9 @@
 import cleanup from 'rollup-plugin-cleanup';
 import json from '@rollup/plugin-json';
 import resolve from '@rollup/plugin-node-resolve';
+import {swc} from 'rollup-plugin-swc3';
 import {terser} from 'rollup-plugin-terser';
-import { readFileSync } from "fs";
```

Comment:
Not sure what eslint is configured to but either keep the spaces here and in [auto/auto.d.ts](https://github.com/chartjs/Chart.js/pull/10595/files#diff-9fe1192344f3acd5427e70ebfb891231a342daea1aa2633367147a15960bd337) or also remove the spaces with all other import brackets.

But giving it a quick look it seems like there are a lot more with spacing as without

Reply:
https://github.com/chartjs/eslint-config-chartjs/blob/master/.eslintrc.yml#L174